docs/link.md

```diff
@@ -0,0 +1,8 @@
+### code-server --link
```

Comment:
since this is a top-level page, this should be a h1 heading

also, since it's a new page, we need to add it to the docs manifest or else it won't be navigable